### PR TITLE
CEDS-2284 Change the implementation to return always 202

### PR DESCRIPTION
**Pull request by PatrykRudnicki**

### CODE REVIEW

File: app/connectors/UpscanConnector.scala [hunk 1]
```diff
@@ -62,9 +64,9 @@ class UpscanConnector @Inject()(conf: AppConfig, wsClient: WSClient)(implicit ec
 
     val filePart = FilePart("file", fileName, Some("text/plain"), Source.single(ByteString(contactDetails.toString)))
 
-    Logger.warn(s"Upload url: ${req.url}")
-    Logger.warn(s"Upload URI: ${req.uri}")
-    Logger.warn(s"Upload Headers: ${req.headers}")
+    logger.debug(s"Upload url: ${req.url}")
```
Comment (PauloPortugal) [inline app/connectors/UpscanConnector.scala]: :+1:

File: app/controllers/UpscanStatusController.scala [hunk 2]
```diff
@@ -179,8 +181,8 @@ class UpscanStatusController @Inject()(
     )
 
   private def getPosition(ref: String, refs: List[String]) = refs match {
-    case head :: tail if head == ref => First(refs.size)
-    case init :+ last if last == ref => Last(refs.size)
+    case head :: _ if head == ref => First(refs.size)
```
Comment (PauloPortugal) [inline app/controllers/UpscanStatusController.scala]: :+1:

File: app/controllers/notification/NotificationCallbackController.scala [hunk 3]
```diff
@@ -32,27 +32,29 @@ class NotificationCallbackController @Inject()(notificationService: Notification
...
 
-  private def saveNotification(notification: NodeSeq) =
+  private def saveNotification(notification: NodeSeq): Future[Status] =
     notificationService.save(notification) map {
       case Right(_) => Accepted
       case Left(e: BadRequestException) =>
-        Logger.error(s"Failed to save invalid notification: $notification", e)
-        BadRequest
+        logger.warn(s"Failed to save invalid notification: $notification", e)
+        Accepted
```
Comment (PauloPortugal) [inline app/controllers/notification/NotificationCallbackController.scala]: :+1:

File: app/connectors/CustomsDeclarationsConnector.scala [hunk 4]
```diff
@@ -31,23 +31,25 @@ import scala.xml.XML
...
+  private val logger = Logger(this.getClass)
+
   private val fileUploadUrl = config.microservice.services.customsDeclarations.batchUploadEndpoint
   private val apiVersion = config.microservice.services.customsDeclarations.apiVersion
   private val clientId = config.developerHubClientId
 
   def requestFileUpload(eori: String, request: FileUploadRequest)(implicit hc: HeaderCarrier): Future[FileUploadResponse] = {
-    Logger.info(s"Request to initiate ${request.toXml}")
-    Logger.info(s"fileUploadUrl: $fileUploadUrl")
+    logger.debug(s"Request to initiate ${request.toXml}")
```
Comment (rashadmughal) [inline app/connectors/CustomsDeclarationsConnector.scala]: Don't change info to debug as once in ET/prod debug will not be available in PROD

**Pull request merged**